### PR TITLE
[feature] Made ui changes to add eraser tool

### PR DESCRIPTION
**Pull request by sisardor**

### CODE REVIEW

**Comment by ZhijieZhang** [comment]
I believe this feature will be included in 5.2.

In that case this PR should be made into branch 5.2, not dev. Dev contains changes in 6.0

File: src/helpers/setupDocViewer.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const addToolsToDocViewer = docViewer => {
   toolModeMap.AnnotationCreateFreeHand2 = new window.Tools.FreeHandCreateTool(docViewer, 'AnnotationCreateFreeHand2');
   toolModeMap.AnnotationCreateFreeHand3 = new window.Tools.FreeHandCreateTool(docViewer, 'AnnotationCreateFreeHand3');
   toolModeMap.AnnotationCreateFreeHand4 = new window.Tools.FreeHandCreateTool(docViewer, 'AnnotationCreateFreeHand4');
+  toolModeMap.AnnotationCreateEraser = new window.Tools.EraserTool(docViewer, 'AnnotationCreateEraser');
```
Comment (ZhijieZhang) [inline src/helpers/setupDocViewer.js]: Prefer doing this in WebViewer instead. Please add this tool in `DocumentViewer.js` ~line 246, `this.toolModeMap`.

The freehand tools and highlight tools above only exist for the purpose of the current UI design

File: src/redux/initialState.js [hunk 2]
```diff
@@ -129,7 +132,8 @@ export default {
       Pan: { showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null },
       AnnotationEdit: { showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null },
       TextSelect: { showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null },
-      MarqueeZoomTool: { dataElement: 'marqueeToolButton', title: 'tool.select', label: 'Marquee Zoom', showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null }
+      MarqueeZoomTool: { dataElement: 'marqueeToolButton', title: 'tool.select', label: 'Marquee Zoom', showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null },
+      AnnotationCreateEraser: { showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: annotation => annotation instanceof window.Annotations.FreeHandAnnotation }
```
Comment (ZhijieZhang) [inline src/redux/initialState.js]: If the eraser tool doesn't have any associated annotations then the value of `annotationCheck` is null. If you are making changes in 5.2 branch, this code is in `map.js`. Let me know if you need any helps with this. Thanks!

File: src/helpers/setupDocViewer.js (hunk 1 above)
Comment (sisardor) [reply src/helpers/setupDocViewer.js]: Ohh ok, I'll do that!

File: src/redux/initialState.js (hunk 2 above)
Comment (sisardor) [reply src/redux/initialState.js]: Ok, I'll update

**Pull request closed without merging**